src/routers/index.js

```diff
@@ -4,4 +4,7 @@ const router = express.Router();
 const userRouter = require("./userRouter")
 router.use("/users", userRouter)
 
-module.exports = router
\ No newline at end of file
+const reviewRouter = require("./reviewRouter")
+router.use("/review", reviewRouter);
```

Comment:
코드 라인 수정해주세요.

reviewRouter를 import 해오는 7번 라인 코드를  userRouter를 import 해오는부분 다음으로 이동시켜 주세요!

마찬가지로 router 모듈에 reviewRouter 미들웨어를 등록하는 코드도 한 데 모아주세요

예시
```suggestion
const userRouter = require("./userRouter")
const reviewRouter = require("./reviewRouter")

router.use("/users", userRouter);
router.use("/review", reviewRouter);
```

Reply:
수정 완료하였습니다.!